examples/typography/index.js

```diff
@@ -1,49 +0,0 @@
-/**
```

Comment:
Why delete this `index.js` file and put it straight into `index.html`? It goes against the principle of Separation of Concerns.

Reply:
I think there are a few reasons:
* it's a demo file, intended to make it easy to follow along. Having to jump between less files is more important than separation of concerns for this use case
* the file already isn't separating concerns. It was embedding style and html in several places in the same file.